ceph_installer/tests/test_util.py

```diff
@@ -102,6 +102,20 @@ def test_redhat_storage_is_false(self):
         }
         assert result == expected
 
+    def test_redhat_use_cdn_is_true(self):
+        data = dict(redhat_storage=True, redhat_use_cdn=True)
+        result = util.get_install_extra_vars(data)
+        assert "ceph_stable_rh_storage" in result
+        assert "ceph_stable_rh_storage_cdn_install" in result
+
+    def test_redhat_use_cdn_is_false(self):
+        data = dict(redhat_storage=True, redhat_use_cdn=False)
+        result = util.get_install_extra_vars(data)
+        assert "ceph_stable_rh_storage" in result
+        assert "ceph_origin" in result
```

Comment:
if `ceph_origin` should be in here, wouldn't the opposite be correct for the other test? Maybe this warrants a separate test for this value